<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_C">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_D">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_E_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_E">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_B">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_A">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_F">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_G">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
